OMCompiler/SimulationRuntime/c/simulation/simulation_runtime.cpp

```diff
@@ -544,14 +544,26 @@ int startNonInteractiveSimulation(int argc, char**argv, DATA* data, threadData_t
   if(omc_flag[FLAG_IIF]) {
     if (omc_flag[FLAG_INPUT_PATH]) {
       const char *tmp_filename;
-      if (0 > GC_asprintf(&tmp_filename, "%s/%s", omc_flagValue[FLAG_INPUT_PATH], omc_flagValue[FLAG_IIF])) {
-        throwStreamPrint(NULL, "simulation_runtime.cpp: Error: can not allocate memory.");
+      struct stat attrib;
```

Comment:
This will not work on Windows. You need to have,

```
#if defined(__MINGW32__) || defined(_MSC_VER)
  struct _stat attrib;
#else
  struct stat attrib;
#endif
```

Reply:
Added it